composer.json

```diff
@@ -14,7 +14,7 @@
         "php": "^7.3",
         "ext-json": "*",
         "keboola/csv": "^1.1",
-        "keboola/db-adapter-snowflake": "1.0.3",
```

Comment:
to se melo releasnout jako minr version, kdyz je tam nova ficura :) coz ti naznaci, ze tady mas mit ^1.1 a ne 1.0.4

Reply:
neřekl bych úplně že to je nová fičura :)